react-native-purchases-ui/ios/PaywallViewWrapper.m

```diff
@@ -36,9 +37,20 @@ - (instancetype)initWithPaywallViewController:(RCPaywallViewController *)paywall
     return self;
 }
 
+
+- (void)reactSetFrame:(CGRect)frame
+{
+    NSLog(@"reactSetFrame: %@", NSStringFromCGRect(frame));
```

Comment:
Should we add a `RNPaywalls` prefix or something so we know this is from us? Same on the other logs

Also, just to make sure, these won't be spamming the log message normally right?

Reply:
Added.

No, it won't spam, they only print on creation, maybe a couple times only